Util/AdminObjectAclManipulator.php

```diff
@@ -271,8 +271,19 @@ protected function buildForm(AdminObjectAclData $data, FormBuilderInterface $for
                     'attr'     => $attr,
                 );
             }
-
-            $formBuilder->add($key, new AclMatrixType(), array('permissions' => $permissions, 'acl_value' => $aclValue));
+            if (3 < Kernel::MAJOR_VERSION) {
```

Comment:
Absolutly not. This is from the Kernel component and we are working with the form one.

@turko Please use `method_exists`.